frontend/packages/erd-core/src/features/erd/components/ERDRenderer/CommandPalette/CommandPalette.module.css

```diff
@@ -6,24 +6,38 @@
   width: 638px;
   border-radius: var(--border-radius-lg);
   box-shadow: 0px 4px 20px 0px var(--shadow-basic-shadow, #000);
-  animation: contentShow 150ms cubic-bezier(0.16, 1, 0.3, 1);
   background-color: var(--pane-background);
   color: var(--global-foreground);
   border: 1px solid var(--global-border);
   z-index: var(--z-index-command-palette);
   font-family: var(--main-font);
 }
 
+.content[data-state='open'] {
+  animation: contentShow 150ms cubic-bezier(0.16, 1, 0.3, 1);
+}
+
+.content[data-state='closed'] {
+  animation: contentHide 100ms cubic-bezier(0.7, 0, 0.84, 0);
+}
```

Comment:
`cubic-bezier(0.7, 0, 0.84, 0)` is a reverse way function of `cubic-bezier(0.16, 1, 0.3, 1)`
Here are good visualizations:

- [cubic-bezier(0.16, 1, 0.3, 1)](https://cubic-bezier.com/#.16,1,.3,1)
- [cubic-bezier(0.7, 0, 0.84, 0)](https://cubic-bezier.com/#.7,0,.87,0)